app/index.html

```diff
@@ -154,10 +162,33 @@ <h2 class="work-item-title primary-font primary-text-color">Multi-Post Stories <
                 </div>
             </div>
         </div>
+
+        <div class="work-item">
```

Comment:
- [ ] This section should consist of 2 rows with 3 projects each.  Please kindly correct it.
- [ ] Each project should contain an image.  Kindly add it.

That way, your project will look a lot more like the design.  💪 

By the way, good job centering those gray boxes 💯

Reply:
Changes done